Resources/views/full/folder.html.twig

```diff
@@ -0,0 +1,43 @@
+{% extends "eZDemoBundle::pagelayout.html.twig" %}
+
+{% block content %}
+    <section class="content-view-full">
+        <div class="class-folder">
+            <div class="row">
+                <div class="col-md-8">
+                    <div class="attribute-header">
+                        <h1>{{  ez_render_field( content, 'name' ) }}</h1>
+                    </div>
+
+                    {% for article in pagerFolder %}
+                        <section class="content-view-children">
+                            {{ render_esi( controller( 'ez_content:viewLocation', { 'locationId': article.contentInfo.mainLocationId, 'viewType': 'line' } ) ) }}
+                        </section>
+                    {% endfor %}
+
+                    {% if pagerFolder.haveToPaginate() %}
+                        <div class="pagination-centered">
+                            {{ pagerfanta( pagerFolder, 'twitter_bootstrap_translated', { 'routeName': location } ) }}
+                        </div>
+                    {% endif %}
+
+                </div>
+
+                <aside class="col-md-4">
+                    <section class="subnavigation row nav-collapse">
+                        <div class="attribute-header">
+                            <h2>{{ ez_render_field( content, 'name' ) }}</h2>
+                        </div>
+
+                        <ul class="break folder-view">
```

Comment:
It would be better to check if the list is empty and only then render `<ul>` tags, otherwise you'll render it empty in that case.
Also it seems the sub-request will render `<ul><ul>`, or am I missing something?